packages/turf-isobands/.npmignore

```diff
@@ -0,0 +1,2 @@
+test
+coverage
```

Comment:
The entire `.npmignore` file is not required, I've changed every module in Turf so all of them have the same syntax & file structure.

Use any turf module as your starting point to propose a new module (except the `index.js`).
https://github.com/Turfjs/turf/tree/master/packages/turf-along